config/site_settings.yml

```diff
@@ -512,7 +512,7 @@ login:
     regex: "^[a-z0-9]+$"
   linkedin_oidc_client_secret:
     default: ""
-    regex: "^[a-zA-Z0-9]+$"
+    regex: "^[a-zA-Z0-9_=\\.]+$"
```

Comment:
Does it have a `\\` char?

Reply:
You need to use double escape because it's in a YAML file. The first one escapes the second in YAML, the second one escapes the dot in the regular expression. (You can search `\\.` in `site_settings.yml` for other examples.